helm/grafana/values.yaml

```diff
@@ -778,22 +977,105 @@ grafana:
   #   requests:
   #     cpu: 50m
   #     memory: 50Mi
-    securityContext: {}
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      seccompProfile:
+        type: RuntimeDefault
     # skipTlsVerify Set to true to skip tls verification for kube api calls
     # skipTlsVerify: true
     enableUniqueFilenames: false
     readinessProbe: {}
     livenessProbe: {}
-    # Log level. Can be one of: DEBUG, INFO, WARN, ERROR, CRITICAL.
-    logLevel: INFO
-    dashboards:
+    # Log level default for all sidecars. Can be one of: DEBUG, INFO, WARN, ERROR, CRITICAL. Defaults to INFO
+    # logLevel: INFO
+    alerts:
```

Comment:
Same as previous comment : do we really need to include this in the values if we're not using it ? It makes the values harder to read IMO